sites/org/pages/[postType]/page/[page].mjs

```diff
@@ -0,0 +1,206 @@
+// Dependencies
```

Comment:
Here too, the shared code seems to add more friction.
Like for one thing, I don't think we need pagination for the blog.

Reply:
Why not? I think uninterrupted flow is always nice, and it costs very little to have it there